src/lib.rs

```diff
@@ -55,6 +55,14 @@ use std::cmp::{Ord, Ordering};
 use std::collections::HashMap;
 use yew::Properties;
 
+#[cfg(feature = "test")]
```

Comment:
Why do we need to keep this behind the "test" feature? Since frontary is a library crate, `alert` won't be linked into the binary unless it is actually used.

Reply:
The below in whole.rs requires it.

```
#[cfg(not(feature = "test"))]
const DEFAULT_NUM_PER_PAGE: usize = 10;
#[cfg(not(feature = "test"))]
const DEFAULT_NUM_PAGES: usize = 10;

#[cfg(feature = "test")]
const DEFAULT_NUM_PER_PAGE: usize = 2;
#[cfg(feature = "test")]
const DEFAULT_NUM_PAGES: usize = 2;
```

Reply:
I don't see how those constants require that `alert` be kept behind "test". Does the following change have anything to do with them?

```diff
diff --git a/src/lib.rs b/src/lib.rs
index 779bba3..3105479 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -55,7 +55,6 @@ use std::cmp::{Ord, Ordering};
 use std::collections::HashMap;
 use yew::Properties;
 
-#[cfg(feature = "test")]
 pub fn alert(msg: &str) {
     web_sys::window()
         .expect("Window should exist")
```

Reply:
I don't want users of frontary to use `alert` except in tests.